src/casp/anndata_to_avro.py

```diff
@@ -324,7 +362,8 @@ def process(input_file, cas_cell_index_start, cas_feature_index_start, avro_pref
     print("Processing core data...")
     # recode the indexes to be the indexes of obs/var or should obs/var include these indices?
     dump_core_matrix(adata.raw.X, row_index_to_cas_cell_index, col_index_to_cas_feature_index, raw_counts_filename)
-        
+
+
```

Comment:
Is this commented block still needed?

Reply:
If I understand this correctly then I think so. I believe this is asking for an ingest-side test to make sure the representations we're creating are consistent with the contents of the AnnData file (there are currently no tests for ingest to parallel the ones you created for extract).